Arrowgene.Ddon.Shared/Model/Enemy.cs

```diff
@@ -68,7 +68,11 @@ public uint GetDroppedExperience()
         {
             return Experience * (NamedEnemyParams.Experience/100);
         }
-  
+
+        public uint GetDroppedPlayPoints()
+        {
+            return GetDroppedExperience()/7500; //TODO: Totally arbitrary. Figure out how to do this properly.
```

Comment:
You can probably find more info searching in Wyrm Hunt. You could also make it so the PP value is loaded from the enemy JSON from a new, optional field, and leave this as a fallback if it's not specified. Either way it can be done on a later PR